controllers/common-controller-utils_test.go

```diff
@@ -61,14 +65,14 @@ func fakeMirrorPeers(manageS3 bool) []multiclusterv1alpha1.MirrorPeer {
 				ManageS3: manageS3,
 				Items: []multiclusterv1alpha1.PeerRef{
 					{
-						ClusterName: TestSourceManagedClusterName,
+						ClusterName: TestSourceManagedClusterName1,
```

Comment:
why this name change? using `1` (integers) for a variable name is not common practise in golang

Reply:
i will change this it something else